cssMain/styleMain.css

```diff
@@ -789,94 +788,41 @@ section .Isaac {
     transform: translateY(100%);
     transition: 2s;
 }
-
-.content p {
-    color: #999;
+.content-m p{
+    color: rgb(230, 233, 229);
 }
-
-
 /*style of the content link*/
-.content a {
+.content-m a{
     width: 100px;
     text-decoration: none;
-    border-radius: 40px;
-    border: 1px solid #fff;
+    border-radius: 20px;
+    border: 5px solid rgb(255, 255, 255);
     display: block;
-    margin: auto;
+    margin: 1px auto;
     padding: 10px;
-    color: white;
-}
+    color: rgb(255, 255, 255);
 
-/*style hover content inside the card*/
-.card:hover .content {
-    transform: translateY(0%);
 }
 
-/*styling of the contact to show hover*/
-.contact {
-    display: flex;
-    color: rgb(15, 12, 12);
-    align-items: center;
-    text-align: center;
+/*style hover content inside the card*/
+.card-m:hover .content-m{
+    transform: translateY(0%)
 }
-
-/*styling of the footer*/
-.footer {
-    background-color: var(--thirdColor);
+/*styling  footer*/
+ .footer-m{ 
+     
     text-align: center;
-    color: #fff;
-    /* height: 3vh; */
-    padding: 20px;
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
+    color: rgb(253, 246, 246);
```

Comment:
Please use VAR for Colors next time. This is almost white and white is our secondary color.